src/index.html

```diff
@@ -4,14 +4,148 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form action="" onsubmit="onSubmit()">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label for="sname">
+            Surname:
+          <input
```

Comment:
```suggestion
        <label for="sname">
          Surname:
          <input
```
Тут дополнительный отступ не нужен